### DISCUSSION TITLE
搜索结果有没有时间优化一下

### DISCUSSION THREAD

**Discussion by droiyan2** [xiaohucode/yidaRule#30] [Ideas]
大佬你好，请问能不能修改一下搜索的逻辑，第一个就是指定搜索的类别(比如小说图片视频分开),还有搜索结果能不能把完全匹配的结果显示在最上方，书源多了实在是不好找，要往下划拉很久

**Comment by xiaohucode** [comment]
搜索输入框右侧有个猫爪图标，可以点击设置过滤结果

**Comment by droiyan2** [comment]
谢谢大佬

**Comment by droiyan2** [comment]
请问除了encodeURI方法还有没有其他的内置解密方法，因为不懂代码，所以挺需要内置这些

**Reply by xiaohucode** [reply]
看看文档https://github.com/xiaohucode/yidaRule/blob/main/Document.md#%E8%A7%84%E5%88%99%E5%86%85%E7%BD%AEcryptojs%E5%8F%AF%E7%9B%B4%E6%8E%A5%E4%BD%BF%E7%94%A8

**Reply by droiyan2** [reply]
谢谢大佬。又有一个新的问题，搜索页获取的详情地址是‘/xs/20966.html’，实际详细章节的地址是‘/list/36750.html’，请问怎样可以把xs替换成list。

**Reply by droiyan2** [reply]
又有一个新问题：请问这个应该怎么解码呀？searchkey=%B6%BC%CA%D0&submit=%CB%D1%CB%F7，网页显示searchkey: （无法解码值）submit: （无法解码值），麻烦大佬了。
